massgov/mayflower-react
# Add generic file icon to download link

Render a generic file icon in download link for acceptable file formats uploadable to mass.gov.
Pass a title prop to svg icons so they can specify this property, particularly pertinent to files grouped as generic.

Testing instructions:
1. npm start
2. go to download link
3. set 'show icon' to true
4. enter href with csv extension (e.g. test.csv) and confirm generic icon is displayed
4. enter href for pdf extension (e.g. test.pdf) and confirm pdf icon is displayed
